### ISSUE TITLE
Shorten arguments and locals

### ISSUE THREAD

**Issue by XVilka** [radare/radare2#12656]
It probably makes sense to shorten local variable names from `local_8h` to `loc_8h` or `var_8h`,
also it makes sense to use `uint8_t` or even `u8`/`ut8` instead of full `unsigned int` or even worse `unsigned long long`.

See how it is too long for some cases:
![screenshot_20190105_184540](https://user-images.githubusercontent.com/203261/50723557-71f1ba80-111a-11e9-987d-79213b47b266.png)

**Comment by sivaramaaa** [comment]
Yup , sgtm , if everyone accepts on single convention then I could change it , just a online change , so @radare any suggestions ?

**Comment by radare** [comment]
short names lgtm, but we must discuss /decide this between all

**Comment by dmknght** [comment]
will radare2 changes it to default or add option and let users choose format (short / full name)?

**Comment by radare** [comment]
this is the place to discuss this. so... what do you think? :D

**Comment by sivaramaaa** [comment]
Okk so the changes would be (according to survey)

```
local_8h ==> var_8h

unsigned int  ===> uint32_t

unsigned long long ====> uint64_t
```

@radare  @XVilka  is there anything else u would like to add to this ?

**Comment by sivaramaaa** [comment]
Hey hi , regarding `uint 8_t` to `u8` many users have voted for uint8_t  that's why ....

![screenshot_20190117-133315__01](https://user-images.githubusercontent.com/18589720/51304003-09310900-1a5d-11e9-8b66-7871c2794140.jpg)

**Comment by XVilka** [comment]
Ok then.

**Comment by dmknght** [comment]
I think create an option `e asm.var.short = False/True` is not very bad. False will show old variable name format and True will show new format.
